### PR TITLE
minor formatting for `odoc`

### PR DESCRIPTION
**Pull request by christinerose**
I've been asked to review and improve the `odoc` documentation, website, and place in the OCaml Manual. Since much of my work is line editing, it can be helpful to make me code owner so I can edit little things like formatting and grammar without having to bother other maintainers and make a lot of noise, if you feel comfortable doing so. I am already code owner on ocaml.org and Dune, among others. So if you find it beneficial, you won't get little formatting changes in a PR again. 

If there are any major changes or suggestions that could alter the meaning, I would always consult with an engineer and open a PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,18 +17,18 @@
   </a>
 </p>
 
-**odoc** is a powerful and flexible documentation generator for OCaml. It reads *doc comments*, demarcated by `(** ... *)`, and transforms them into a variety of output formats, including HTML, LaTeX, and man pages.
+**`odoc`** is a powerful and flexible documentation generator for OCaml. It reads *doc comments*, demarcated by `(** ... *)`, and transforms them into a variety of output formats, including HTML, LaTeX, and man pages.
```
Comment (Julow) [inline README.md:20]: `odoc` is the name of the command-line tool but Odoc is the name of the project ? I think the README never mention the command-line tool.

**Review by Julow — COMMENTED** [review]
I think we don't have the concept of "code owners" on Odoc. Do you mean to ask for commit access ?

**Comment by christinerose** [comment]
Yes! (Forgive my ignorance in the terminology!) Something that enables me to commit without having to make a PR.

File: README.md (hunk 1 above)
Comment (christinerose) [reply README.md:20]: I had a lengthy conversation with Jon a few years ago to hammer out the official name for `odoc‘, and I had suggested both ODoc and Odoc, but he wanted the official name to be `odoc‘.

**Review by panglesd — APPROVED** [review]
I have no opinion on odoc's name, but other line editing are improvements.

I don't mind giving you push access at all, but it might be good to still open line editing PR, and merge them yourself directly (it gives me more visibility to follow what changes happen in the docs, eg that I should write "`odoc`"; not pushing on `main`/`master` might be safer against git mistakes?)

**Comment by christinerose** [comment]
That sounds perfect!

**Comment by christinerose** [comment]
If you're happy with these changes, @panglesd, do you want to merge them? I don't yet have write access.

**Comment by panglesd** [comment]
I was waiting to see if @jonludlam confirmed that `odoc` is the name to use for this project...

**Comment by dbuenzli** [comment]
Not sure why we need to wait on @jonludlam here. For as far as I can remember this project has always been known as odoc and there's little point in multiplying names especially if it's only to make them differ by capitals which 95% of users won't bother about when discussing the software.

**Comment by panglesd** [comment]
I did not know about the codoc name.

When I read "lengthy discussion" in a github thread, I start to be paralyzed by fear and merging becomes a heroic act.

**Pull request merged**